pkgs/servers/ankisyncd/default.nix

```diff
@@ -1,68 +1,58 @@
-{ lib
-, fetchFromGitHub
-, python3
-}:
+{ lib, stdenvNoCC, fetchFromGitHub, rustPlatform, protobuf }:
 
-python3.pkgs.buildPythonApplication rec {
+let
   pname = "ankisyncd";
```

Comment:
Perhaps

```suggestion
  pname = "ankisyncd-rs";
```

Reply:
Up to you though.

Reply:
the original name for either the old python version and new rust ones are anki-sync-server and anki-sync-server-rs which aren't the same, and both provide the 'ankisyncd' binary, so I'm not sure honestly...

Could you remind me what pname is used for? nix shell and similar will use whatever is defined in all-package's toplevel, so pname is for listing ouputs?

Reply:
It should be called are anki-sync-server and are anki-sync-server-rs respectively then.

pname is purely for informational purposes; for us humans to interpret.